lib/ecmascript.ts

```diff
@@ -4785,16 +4754,10 @@ export function RoundDuration(
       const oneYear = new TemporalDuration(days < 0 ? -1 : 1);
       // This conversion is needed because of an issue in the spec that's likely to change.
       // See https://github.com/tc39/proposal-temporal/issues/1821.
-      relativeTo = CreateTemporalDateTime(
+      relativeTo = CreateTemporalDate(
         GetSlot(relativeToDateOnly, ISO_YEAR),
         GetSlot(relativeToDateOnly, ISO_MONTH),
         GetSlot(relativeToDateOnly, ISO_DAY),
-        0,
-        0,
-        0,
-        0,
-        0,
-        0,
         GetSlot(relativeTo, CALENDAR)
       );
       let { days: oneYearDays } = MoveRelativeDate(calendar, relativeTo, oneYear);
```

Comment:
If you want to remove the conversion above, then this line needs be changed too:

```suggestion
      let { days: oneYearDays } = MoveRelativeDate(calendar, relativeToDateOnly, oneYear);
```

This is a good lesson for the future to be extra-careful when copying code from proposal-temporal!  😄

Reply:
Definitely - though, as I mentioned above, this is actually the right content for this line if the reassignment was there.